dashboard/app/helpers/users_helper.rb

```diff
@@ -116,4 +116,15 @@ def level_info(user, script_level, user_levels)
       0
     end
   end
+
+  def options_for_user_enrollment_users
+    options = []
+
+    User.all.each do |course|
```

Comment:
I don't know if the admin column is indexed so that might be :crying_cat_face: too. And it looks like you copy/pasted the proc from course and meant to do `|user|` here